server/models/Expense.js

```diff
@@ -16,32 +16,44 @@ module.exports = function (Sequelize, DataTypes) {
       references: 'Users',
       referencesKey: 'id',
       onDelete: 'SET NULL',
-      onUpdate: 'CASCADE'
+      onUpdate: 'CASCADE',
+      allowNull: false
     },
 
     GroupId: {
       type: DataTypes.INTEGER,
       references: 'Groups',
       referencesKey: 'id',
       onDelete: 'SET NULL',
-      onUpdate: 'CASCADE'
+      onUpdate: 'CASCADE',
+      allowNull: false
     },
 
     currency: {
       type: DataTypes.STRING,
       defaultValue: 'USD',
+      allowNull: false,
       set(val) {
         if (val && val.toUpperCase) {
           this.setDataValue('currency', val.toUpperCase());
         }
       }
     },
 
-    amount: DataTypes.INTEGER,
-    title: DataTypes.STRING,
-    description: DataTypes.TEXT,
```

Comment:
Is there a reason to remove description? I think we'll need two fields: one short like `title` and one long for comments, optional notes, etc. `notes` maybe?